generate/templates/templates/struct_content.cc

```diff
@@ -123,6 +123,17 @@ NAN_METHOD({{ cppClassName }}::JSNewFunction) {
 
   instance->Wrap(info.This());
 
+  // Set default uninitialized property values.
+  {% each fields as field %}
+    {% if not field.ignore %}
+    {% if not field | isPayload %}
+      {% if field.cppClassName == "String" %}
```

Comment:
Is there a more generic way I should handle this? I'd like to ensure this fixes problems for all primitive values. Maybe just Strings and Numbers for now?